k8s/jobs/mediawikiUpdate.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: mediawiki-update-
+  namespace: adhoc-jobs
+spec:
+  ttlSecondsAfterFinished: 86400
+  backoffLimit: 0
+  template:
+    metadata:
+      name: mediawiki-update
+    spec:
+      containers:
+        - name: mediawiki-update
+          command:
+            - bash
+            - -c
+            - |
+              set -euo pipefail
+              export MW_INSTALL_PATH=/var/www/html/w/
+              php /var/www/html/w/maintenance/update.php \
+                --skip-config-validation \
+                --skip-external-dependencies \
+                --quick \
```

Comment:
We should check if these are the right flags; @dati18 found them as recommended from google. We should check if it's what we did before in the API: https://github.com/wbstack/mediawiki/blob/main/dist-persist/wbstack/src/Internal/PreApiWbStackUpdate.php

In the past we just ran `--update` perhaps that's all we want.

Reply:
https://www.mediawiki.org/wiki/Manual:Update.php

Reply:
I wonder also if this is a sign we need to tighten our CI in the mediawiki repo to ensure the lock file is up to date?

Reply:
This is now removed because it's not an issue on the 143 image